src/ui/FileViewer/ToggleHeader/Title/Title.tsx

```diff
@@ -1,53 +1,64 @@
-import cs from 'classnames'
 import eq from 'lodash/eq'
 import isUndefined from 'lodash/isUndefined'
-import PropTypes from 'prop-types'
 import { useState } from 'react'
 
 import { useLocationParams } from 'services/navigation'
 import { useRepoBackfilled, useRepoFlagsSelect } from 'services/repo'
+import { cn } from 'shared/utils/cn'
 import Icon from 'ui/Icon'
 import MultiSelect from 'ui/MultiSelect'
 
-import CoverageSelect from './CoverageSelect'
+import { CoverageSelect } from './CoverageSelect'
 
-export default function Title({ title, children, sticky = false }) {
+interface TitleProps {
+  title?: React.ReactNode
+  children?: React.ReactNode
+  sticky?: boolean
+}
+
+function Title({ title, children, sticky = false }: TitleProps) {
   return (
     <div
       data-testid="title-wrapper-div"
-      className={cs(
-        { 'z-10 sticky top-[4.5rem]': sticky },
-        'flex flex-col sm:flex-row items-start sm:items-center justify-between gap-4 flex-wrap bg-white px-0 w-full lg:w-auto flex-1 lg:flex-none md:mb-1'
+      className={cn(
+        'flex flex-col sm:flex-row items-start sm:items-center justify-between gap-4 flex-wrap bg-white px-0 w-full lg:w-auto flex-1 lg:flex-none md:mb-1',
+        { 'z-10 sticky top-[4.5rem]': sticky }
```

Comment:
was swapping the ordering to follow a pattern like conditionals come at the end since they don't overlap?

Reply:
This was really just a stylistic change ... default styles should go first, and any "overriding" styles second. We want to do this cause our internal `cn` function uses `tailwind-merge` under the hood, which will remove any "overlapping" or "conflicting" classnames and give preference to those later in the string or argument array.